pkg/providers/aws/provider_redis.go

```diff
@@ -131,6 +131,20 @@ func (p *RedisProvider) CreateRedis(ctx context.Context, r *v1alpha1.Redis) (*pr
 		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
 	}
 
+	updatesAllowed, err := resources.VerifyRedisUpdatesAllowed(ctx, p.Client, r.Namespace, r.Name)
+	if err != nil {
+		msg := "failed to verify if updates are allowed"
```

Comment:
message test, - could we add "Redis"? , like ""failed to verify if Redis updates are allowed" (check also for Postgres). Make errors Info more specific, to make debugging/investigation easier.

Reply:
Agreed, addressed in new commit